packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/AdvancedCoinSettingsModal/AdvancedCoinSettingsModal.tsx

```diff
@@ -40,11 +39,7 @@ interface AdvancedCoinSettingsModalProps {
 
 export const AdvancedCoinSettingsModal = ({ coin, onCancel }: AdvancedCoinSettingsModalProps) => {
     const blockchain = useSelector(state => state.wallet.blockchain);
-    const network = networksCompatibility.find(network => network.symbol === coin);
```

Comment:
Although we are looking at `features`, which can be different for different account, this previous implementation found the first network, which was the `normal` account